app/pages/notifications/comment.cjsx

```diff
@@ -19,6 +19,7 @@ module.exports = React.createClass
       <div className="talk-comment">
         <Comment
           data={@props.data.comment}
+          delivered={@props.notification.delivered}
```

Comment:
It's pretty awkward that 'talk/search-result' is displaying comment notifications.

It's even more awkward to change the search-result to track notification status.

It's probably time to break this out.